### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RoyRao2333**
Update my submission.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ To add your own project below, just [edit](https://github.com/wwdc/2021/edit/mas
 |[Jose Adolfo Talactac](https://twitter.com/devjoseadolfo)|[GitHub](https://github.com/devjoseadolfo/CircuitPlay)|[YouTube](https://youtu.be/pm3mlDZJSes)|SwiftUI, Accelerate, RealityKit, ARKit|Submitted|
 |[Niklas Bülow](https://twitter.com/NiklasBuelow)|[GitHub](https://github.com/insightmind/WWDC21SwiftStudentChallenge)||SpriteKit, UIKit, AVFoundation|Submitted|
 |[Oscar Gorog](https://github.com/OscarGorog)|[GitHub](https://github.com/OscarGorog/WWDC21-Playground)||SwiftUI, Combine, RealityKit, AVFoundation|Submitted|
+|[Roy Rao](https://github.com/RoyRao2333)|[GitHub](https://github.com/RoyRao2333/WWDC21-Apple-Scholarship)||Cocoa, SwiftUI, Combine, AVKit, PlaygroundSupport|Submitted|
```
Comment (thisIsTheFoxe) [inline README.md:29]:
```suggestion
|[Roy Rao](https://github.com/RoyRao2333)|[GitHub](https://github.com/RoyRao2333/WWDC21-Apple-Scholarship)||Cocoa, SwiftUI, Combine, AVKit|Submitted|
```

**Review by thisIsTheFoxe — CHANGES_REQUESTED** [review]
Thanks for your submission. Can you just remove `PlaygroundSupport` from Technologies? Since everyone's using it we don't include it in the list :)

**Comment by RoyRao2333** [comment]
> Thanks for your submission. Can you just remove `PlaygroundSupport` from Technologies? Since everyone's using it we don't include it in the list :)

sure thing! Changing it right away~

**Review by thisIsTheFoxe — APPROVED** [review]
Great! Thank you and good luck 🎉

**Pull request merged**